src/main/java/seedu/duke/reflection/ReflectUi.java

```diff
@@ -0,0 +1,28 @@
+package seedu.duke.reflection;
+
+import seedu.duke.ui.TextUi;
+
+/**
+ * This section is to be updated with main UI class
+ */
+public class ReflectUi extends TextUi {
+    private static final String SEPARATOR = "=";
+
+    /**
+     * Call setSeparator() method inherited from TextUi superclass to re-define separator.
+     */
+    public ReflectUi() {
+        setSeparator(SEPARATOR);
+    }
+
+    /**
+     * Print greeting logo.
+     *
+     * @param logo Greeting logo
+     */
+    public void printLogo(String logo) {
+        printSeparator();
+        System.out.print(logo);
```

Comment:
i create a method for ```System.out.print(logo)``` then call this method instead, is this what you want?

Reply:
Oh ok this is complicated that'll be an unnecessary abstraction instead. Let's just leave it as is first, but think of another way meanwhile. Might get flagged during PE.

Reply:
@nichyjt @YongbinWang @BernardLesley any ideas?

Reply:
A key reason why we decided to have the `TextUi` class is to abstract away `System.out.*` method calls. 

I think that if a method exists to abstract away a certain function (such as printing a message), we should use that method rather than reinvent the wheel. 

Because of that, I think that the call to `System.out` here is in violation of SLAP and makes the code less cohesive.

@wenxin-c Is there no way to print the logo using existing methods within the superclass?

Reply:
but I tot it's okay to add stuff in subclasses?? cause some behaviours might varry across subclasses?